frequently-asked-questions.md

```diff
@@ -0,0 +1,90 @@
+
+**Q: I see a lot of projects have been archived. Which projects are active right now?**
+A: We have several active projects going on and need work: Mentorship System ([Android](https://github.com/anitab-org/mentorship-android), [Backend](https://github.com/anitab-org/mentorship-backend), [Flutter](https://github.com/anitab-org/mentorship-flutter), [iOS](https://github.com/anitab-org/mentorship-ios)), [anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io), BridgeInTech ([backend](https://github.com/anitab-org/bridge-in-tech-backend) and [web](https://github.com/anitab-org/bridge-in-tech-web)), [VMS](https://github.com/anitab-org/vms), [Portal](https://github.com/anitab-org/portal), Open Source Programs ([web](https://github.com/anitab-org/open-source-programs-web) and [backend](https://github.com/anitab-org/open-source-programs-backend)). If you want to work on an archived project, you can fork it and work on it independently. Your contributions will still be recognised. 
+
+**Q: I picked out a project. What do I do next?**
+A: Each project comes with a README which guides you on how to setup the project. If you face any problems you need to head over to the project’s Zulip channel and ask your question there. Someone from the community will help you out. If you need help right away, please mention @project-managers and someone will help you out. Once the project is setup, read the contributing guidelines to make your first contribution.
+
+## Open Source Programs
+
+**Q: Can previous ambassadors, apply to be an ambassador again?**
+A: Yes! You can re-apply.
+
+**Q: Do GSoC mentors receive compensation?**
+A: Please email gsoc-support@google.com for inquiries and learn more information on the [Google Summer of Code official website](https://summerofcode.withgoogle.com/).
+
+**Q: Can I contribute to GSoC projects even if I’m not a GSoC student?**
+A: Yes, GSoC projects are discussed in public channels because we want community involvement. You can even attend weekly meetings of each project if you want to! These projects are just as open source as they were before the GSoC selections. You can contribute with ideas, and even issues if they're not labeled as GSoC issues.
+
```

Comment:
Here 'labeled' is used whereas on line number 37 'labelled' is used. Both are right 'labeled' is used in American English, and 'labelled' is used in British English.

Reply:
have chosen labeled :) thank you so much for pointing that out